include/gzip/compress.hpp

```diff
@@ -1,8 +1,10 @@
-#include <iostream>
+#include <gzip/config.hpp>
+
 // zlib
 #include <zlib.h>
 // std
 #include <limits>
+#include <string>
```

Comment:
Looks like `make format` tests are failing due to alphabetical order here .. just need to move below `stdexcept`

Reply:
Ah, thank you for noticing - fixed in 912a7f8